### PR TITLE
navigation bottom added to client list

### PR DESCRIPTION
**Pull request by abdullahcalisir12**
# Issue Being Addressed
#54 
Put the issue(s) # that your PR is addressing here. e.g. #15, #29

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For Bug Fixes:** What was the root cause? How do your changes fix the bug effectively?

**For Refactors:** What are we refactoring and why? How do your changes address this need for refactoring?

**For New Features:** What feature are we adding? Explain your technical approach to doing this.
Navigation bottom added
**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.

**For Other:** State what 'type' of PR yours is and elaborate on the PR's technical content.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1.Go to https://deploy-preview-132--browtricks.netlify.app/tenant/{tenantSlug}/clients
2. Change tenant slug

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

**Comment by paulchrisluke** [comment]
I'm unable to review this component, can't progress past making a client. Made backend issue for error messaging on add client, but this issues still needs to be reviewed when ready https://github.com/whynotearth/meredith-core/issues/245

**Comment by paulchrisluke** [comment]
unable to progress past adding client: 

`TypeError: Cannot read property '0' of null`
[deploy-preview-132--browtricks.netlify.app-1595295015605.log](https://github.com/whynotearth/BrowTricks/files/4950764/deploy-preview-132--browtricks.netlify.app-1595295015605.log)

**Comment by paulchrisluke** [comment]
image
should be transparent

File: src/components/BaseNavigationBottom.vue [hunk 1]
```diff
@@ -0,0 +1,181 @@
+<template>
+  <nav class="flex w-full fixed bottom-0 bg-transparent z-100" id="bottom-nav">
```
Comment (mort3za) [inline src/components/BaseNavigationBottom.vue:2]: `id` is not in use, remove please.

**Comment by mort3za** [comment]
image

In 1920px width, container not working.

File: src/views/ClientList.vue [hunk 2]
```diff
@@ -8,18 +8,21 @@
...
           v-if="
             key === 0
               ? true
-              : client.firstName[0].toUpperCase() !=
-                clients[key - 1].firstName[0].toUpperCase()
+              : client.firstName &&
+                clients[key - 1] &&
+                clients[key - 1].firstName &&
+                client.firstName[0].toUpperCase() !==
+                  clients[key - 1].firstName[0].toUpperCase()
```
Comment (mort3za) [inline src/views/ClientList.vue:21]: Move these logics to a method for readability.

File: src/components/BaseNavigationBottom.vue [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+</script>
+
+<style scoped>
+.toggle-button {
+  top: -30px;
+  right: 16px;
+}
+
+.toggle-button:hover {
+  background: #03b3f9;
```
Comment (mort3za) [inline src/components/BaseNavigationBottom.vue:117]: These styles need a cleanup, use tailwind features and variables please.

**Comment by mort3za** [comment]
In Chrome Android, when I tap on the FAB button, for a second it becomes fully white.

**Pull request merged**